hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/util/DistCpUtils.java

```diff
@@ -592,10 +592,14 @@ public static CopyMapper.ChecksumComparison checksumsAreEqual(
     // comparison that took place and return not compatible.
     // else if matched, return compatible with the matched result.
     if (sourceChecksum == null || targetChecksum == null) {
+      LOG.error("Checksum incompatible. Source checksum: {}, target checksum: {}",
```

Comment:
this is not unusual against object stores, as all stores which don't have hdfs-+compatible checksums disable them so that distcp to cloud stores don't blow up. If you logged at error then there'd be an entry for every single copy.

propose: use a LogExactlyOnce at info to day source or target fs doesn't support checksums and that they should use -skipCrc